deployability/modules/testing/tests/helpers/executor.py

```diff
@@ -0,0 +1,101 @@
+import json
+import requests
+import subprocess
+import urllib3
+import yaml
+
+from base64 import b64encode
+
+
+class Executor:
+
+    @staticmethod
+    def execute_command(inventory_path, command) -> str:
```

Comment:
Why the separation into 2 methods that practically do the same thing? Wouldn't it be feasible to unify them? or that one calls the other

Reply:
Yes but you can do something like this:

```
@staticmethod
    def execute_commands(inventory_path, commands=[]) -> dict:

        with open(inventory_path, 'r') as yaml_file:
            inventory_data = yaml.safe_load(yaml_file)

        host = inventory_data.get('ansible_host')
        port = inventory_data.get('ansible_port')
        private_key_path = inventory_data.get('ansible_ssh_private_key_file')
        username = inventory_data.get('ansible_user')

        results = {}
        for command in commands:
            results[command] = execute_command(inventory_path, command):

        return results
```

So you reuse the execute_command() method in the execute_commands()